### PR TITLE
[#1268] Disable case document button when form is being submitted

### PR DESCRIPTION
**Pull request by vaszig**

### CODE REVIEW

**Review by Bartvaderkin — COMMENTED** [review]
Seems reasonable, maybe it should change style to show disabled state?

**Comment by jiromaykin** [comment]
How can I test this in the front-end? Do I need to add a new Aanvraag through Open Forms first?

**Comment by jiromaykin** [comment]
How can I test this in the front-end? Do I need to add a new Aanvraag through Open Forms first?

File: src/open_inwoner/js/components/cases/index.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+class DisableSubmitButton {
+  constructor(form) {
+    this.form = form
+    this.form.addEventListener('submit', this.disableButton.bind(this))
+  }
+
+  disableButton() {
+    const submitButton = this.form.querySelector('button[type="submit"]')
+    submitButton.setAttribute('disabled', 'true')
```
Comment (jiromaykin) [inline src/open_inwoner/js/components/cases/index.js:9]: in the Cases styles you could add something similar to this:

`.button[type="submit"]:disabled {
    background-color: var(--color-primary-darker);
    color: var(--color-gray)
  }`

**Review by jiromaykin — CHANGES_REQUESTED** [review]
I suggested something that needs to be checked if it works, maybe you'll need a more specific selector for the disabled button inside a form.

File: src/open_inwoner/js/components/cases/index.js (hunk 1 above)
Comment (vaszig) [reply src/open_inwoner/js/components/cases/index.js:9]: Thanks for the hint. I added something similar using the same pattern we use for other disabled buttons in the project.

**Review by jiromaykin — APPROVED** [review]

**Pull request merged**